src/Concerns/Exportable.php

```diff
@@ -73,6 +74,10 @@ public function queue(string $filePath = null, string $disk = null, string $writ
             throw NoFilePathGivenException::export();
         }
 
+        if ($this instanceof FromCollection) {
```

Comment:
FromArray and FromView could be queued in similar fashion

Reply:
I make a pull request which completely rewrites the export queueing.